include/crm/common/results.h

```diff
@@ -125,7 +125,10 @@ enum pcmk_rc_e {
     pcmk_rc_ipc_pid_only        = -1020,
     pcmk_rc_ipc_unresponsive    = -1019,
     pcmk_rc_ipc_unauthorized    = -1018,
+
+    //! \deprecated This value will be removed in a future release
     pcmk_rc_no_quorum           = -1017,
+
```

Comment:
Fair enough. It's not a big deal since it's probably less than 10 lines altogether to support this RC. I just figured if we weren't using it it was clutter, and could be added back later.

The flip side is that it doesn't look great to deprecate something and un-deprecate it later I suppose.